charts/rucio-daemons/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: rucio-daemons
-version: 36.0.0
+version: 36.0.1
```

```suggestion
version: 36.0.2
```

@Soap2G since PR#227 was merged I think you need to update this version number (CI passed before the other PR was merged).